src/workerd/api/tests/js-rpc-test.js

```diff
@@ -259,6 +268,25 @@ export class MyService extends WorkerEntrypoint {
       abort() { ctx.abort(new RangeError("foo bar abort reason")); }
     };
   }
+
+  async writeToStream(stream) {
+    let writer = stream.getWriter();
+    let enc = new TextEncoder();
+    await writer.write(enc.encode("foo, "));
+    await writer.write(enc.encode("bar, "));
+    await writer.write(enc.encode("baz!"));
+    await writer.close();
+  }
+
+  async readFromStream(stream) {
+    return await new Response(stream).text();
+  }
+
+  async returnReadableStream() {
+    let { readable, writable } = new IdentityTransformStream();
+    this.ctx.waitUntil(this.writeToStream(writable));
+    return readable;
```

Comment:
We need a test that verifies that a `readable` that is already locked and disturbed cannot be passed.

Reply:
4f34626f